build/AzurePipelinesTemplates/MUX-NugetReleaseTest-Job.yml

```diff
@@ -89,3 +101,11 @@ jobs:
       solutionPath: $(solutionPath)
       nugetConfigPath: $(nugetConfigPath)
 
+  - template: MUX-PublishDevProject-Steps.yml
```

Comment:
>MUX-PublishDevProject-Steps.yml [](start = 14, length = 31)

If that yml file works for publishing pretty much any project then maybe rename the file to MUX-PublishProjectOutput-Steps.yml or something like that?